.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,72 @@
+name: Bug Report
+description: File a bug report
+title: '[Bug] '
+labels: ["c.Bug 🐛", "s.ToInvestigate"]
```

Comment:
Minor thing, not sure if `ToInvestigate` should be the default tag for bug reports since this would be changing the current issues workflow as we'll need to purposefully remove the tag. This seems to be conflating `ToInvestigate` with a "needs triage" kind of tag.

Reply:
> issues workflow

What workflow are you referring to? 

> This seems to be conflating ToInvestigate with a "needs triage" kind of tag.

Indeed I intended to add a "needs triage" label and the closest label that we are currently using is "s.ToInvestigate". Perhaps can others can chip in on this as well @MarkBind/active-devs

Reply:
> What workflow are you referring to?

Oh I meant the general sequence of actions maintainers take in triaging/managing the issue tracker 😅 Not the Github workflows

Reply:
> > What workflow are you referring to?
> 
> Oh I meant the general sequence of actions maintainers take in triaging/managing the issue tracker 😅 Not the Github workflows

Yup, I am trying to automate that need to manually tag it as "to investigate" :) Unless it's specified for MarkBind devs to follow a workflow that says maintainers must triage and manually apply tags?

Reply:
I think it's that I see most to investigate tags show up on issues where there are difficulties in reproducing the workflows... perhaps a senior developer could weigh in?

Reply:
Thanks for the feedback. I have removed the tag.